client/locations-list-map.js

```diff
@@ -1528,6 +1536,9 @@ function initMapSearch(data, filters) {
     e.preventDefault();
     resetMap(data, filters);
     $search.val('');
+    Object.keys(filterSelectors).forEach((current) => {
```

Comment:
FYI, you can use Object.entries() to avoid the second lookup through the data structure.

Reply:
Gotcha, I think I steered away from the methods that did not have IE11 compatibility when writing the code.

Reply:
ah yes! Good thinking!  Though in our code, we've been relying on babel and polyfills to handle this gap. Steve fixed some mistakes we had so it should be safe to use whatever.